aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -44,12 +44,10 @@ Tensor isclose(const Tensor& self, const Tensor& other, double rtol, double atol
 
   auto close = actual_error <= max_error;
 
-  if (isFloatingType(self.scalar_type()) && isFloatingType(other.scalar_type())) {
+  if (isFloatingType(self.scalar_type()) && isFloatingType(other.scalar_type())
+  || isComplexType(self.scalar_type()) && isComplexType(other.scalar_type())) {
```

Comment:
hm, does `isclose` work for complex now? I thought @anjali411 said == is not yet enabled for complex (and same for != a few lines down). Also, since TORCH_CHECK a few lines above makes sure that `self` and `other` types are the same, this line works too hard.

Reply:
ha! `torch.equal` doesn't work for complex so I thought `==` wouldn't either, but it does

Reply:
I tried using `==` and `!=` for complex tensors on the devel branch and it does seem to work. I'll make the necessary updates @ngimel.